### PR TITLE
Add Docker for Development

### PR DESCRIPTION
**Pull request by andreirtaylor**
This is not done yet we still need to add in the docker stuff for deployment. But this will let us develop easier.

### CODE REVIEW

File: config/index.js.example [hunk 1]
```diff
@@ -8,9 +8,14 @@ config.redisStore = {
...
 }
 
+config.mongo = {
+  url: process.env.MONGO_URL,
+}
+
+
 config.google = {
-  clientID: "Your google client ID here",
-  clientSecret: "client secret here",
```
Comment (cpjk) [inline config/index.js.example:13]: :+1:

**Comment by cpjk** [comment]
does this work yet?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,12 @@
...
 
-You will also need  a running MongoDB instance, with a database named 'choralweb'.
-
-```
-git clone git@github.com:ChoralCloud/ChoralWeb.git \
-&& cd ChoralWeb \
-&& npm install
-```
-
-You will need to creat a new config file based on the template:
```
Comment (andreirtaylor) [inline README.md]: Shit I didnt mean to delete this

File: config/index.js.example (hunk 1 above)
Comment (andreirtaylor) [reply config/index.js.example:13]: This will cause the config to error on this line istead of trying to continue

**Review by andreirtaylor — COMMENTED** [review]
What do you mean?

**Comment by cpjk** [comment]
> What do you mean?

Like does docker compose work with this? I didn't try it

**Comment by cpjk** [comment]
I don't want to do any funky shit unless it's already working

**Comment by andreirtaylor** [comment]
Yeah it works

**Comment by jboretsky** [comment]
I just tested on your branch, it seems to be working so I'm cool with merging it if Chris is.

**Comment by cpjk** [comment]
sry just finishing up some mongo stuff then i'll get on it

**Pull request merged**